lib/modules/apostrophe-notifications/views/notification.html

```diff
@@ -0,0 +1,3 @@
+<div class="apos-notification apos-notification--hidden" data-apos-notification>
+  <p class="apos-notification-message" data-apos-notification-message>{{ data.message }}</p>
```

Comment:
In index.js I already interpolated them, and I pass the prepared string as data.message. I do it this way because I need to use .apply to pass a prepared array of arguments.